deployment/k8s/charts/components/migrationConsole/values.yaml

```diff
@@ -1,3 +1,4 @@
+developerModeEnabled: false
```

Comment:
What do you think about not having this flag and we always deploy all the tests systems?

Reply:
See this section above
```
{{- if $developerModeEnabled  }}
        - name: local-console-libs
          hostPath:
            path: /opensearch-migrations/TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib
            type: Directory
```

That mounts the LOCAL HOST directory from the developer's machine into the running container.